### ISSUE TITLE
Featured Image Section missing in Editor for multiple themes

### ISSUE THREAD

**Issue by villanovachile** [Automattic/wp-calypso#54943]
### Steps to reproduce the behavior

The featured image section is missing using Gutenberg on Simple Sites. Issue is not present on Atomic sites, nor when using the Classic Editor.

Work around for now will be to use the Classic Editor to set a featured image.

#### Is this specific to the applied theme? Which one?
Affected themes so far:
• Dicot
• Floral
• Patch
• Tortuga
• Connect
• Veni

#### Does this happen on simple or atomic sites or both?
• Simple

#### Is there any console output or error text?
![image](https://user-images.githubusercontent.com/65082164/127223925-4540bce9-d18e-4f6b-b9b5-d3770f84fa02.png)

#### Level of impact (Does it block purchases? Does it affect more than just one site?)
Quite high, as it affects multiple themes

**Comment by aamills** [comment]
Ran into this issue myself, with a user in 30840456-hc, using the Veni theme. Guided them to Classic to add/edit featured images for now.

**Comment by villanovachile** [comment]
Report in 4178605-zen using the Dicot theme.

**Comment by metabreakr** [comment]
30840165-hc / 4178686-zen

**Comment by allilevine** [comment]
@villanovachile I wasn't able to reproduce this issue, so I'm hoping it was fixed! Are you still seeing it on Simple Sites?

My testing steps:
- Activate one of the listed themes on a Premium Simple Site
- Create a new post
- Check the editor sidebar for the featured image section
- Add a featured image

<img width="287" alt="Screen Shot 2021-08-03 at 4 05 28 PM" src="https://user-images.githubusercontent.com/1689238/128078896-3ed94a17-7974-4f6d-9842-a9dd57391bc3.png">


I do still see a console error related to `window.getComputedStyle`, but not the `theme_supports TypeError`.

I tested in Firefox and Chrome.
